package.json

```diff
@@ -2,6 +2,10 @@
   "name": "effect",
   "version": "2.0.0-next.58",
   "type": "module",
+  "exports": {
```

Comment:
Was leaving this in intentional? I think @fubhy had removed it purposefully.

Reply:
No it wasn't :)